lib/errors.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+
+const getContent = (err, debug, translate) => {
+  var content = {};
+
+  if (err.code === 'SESSION_TIMEOUT') {
+    content.title = translate('errors.session.title');
```

Comment:
Perhaps we should provide our own very tiny translate lib that just converts the keys to English? That way, the translate would be optional like the logger.